--- zaz-0.3.1/extra/zaz.desktop~	2009-11-02 22:50:40.000000000 +0000
+++ zaz-0.3.1/extra/zaz.desktop	2010-01-29 22:44:29.000000000 +0000
@@ -1,10 +1,10 @@
 [Desktop Entry]
-Version=1.0
 Type=Application
 Name=Zaz
-Comment=Zaz ain't Z***
+Comment=An arcade action puzzle game
+Comment[pl]=Zręcznościowa gra logiczna
 TryExec=zaz
 Exec=zaz
-Categories=Game
+Categories=Game;ArcadeGame;
 Icon=zaz
-MimeType=
+# vi: encoding=utf-8
