--- zaz-0.3.3/extra/zaz.desktop~	2010-02-20 03:04:53.000000000 +0100
+++ zaz-0.3.3/extra/zaz.desktop	2010-02-22 10:29:57.895696681 +0100
@@ -1,12 +1,12 @@
 [Desktop Entry]
-Version=1.0
 Type=Application
 Name=Zaz
 Name[ru]=Zaz
 Comment=Zaz ain't Z***
+Comment[pl]=Zręcznościowa gra logiczna
 Comment[ru]=Zaz не клон Z***
 TryExec=zaz
 Exec=zaz
-Categories=Game
+Categories=Game;ArcadeGame;
 Icon=zaz
-MimeType=
+# vi: encoding=utf-8
