--- zaz-0.3.0/extra/zaz.desktop~	2009-09-04 22:30:51.000000000 +0200
+++ zaz-0.3.0/extra/zaz.desktop	2009-10-07 15:08:07.000000000 +0200
@@ -1,10 +1,10 @@
 [Desktop Entry]
-Version=1.0
 Type=Application
 Name=Zaz
-Comment=Zaz ain't Z***
-TryExec=zaz
+Comment=An arcade action puzzle game
+Comment[pl]=Zręcznościowa gra logiczna
 Exec=zaz
 Categories=Game;LogicGame;
 Icon=zaz
-MimeType=
+# vi: encoding=utf-8
+
