--- zaz-0.8.0/extra/zaz.desktop~	2010-05-09 18:25:22.000000000 +0200
+++ zaz-0.8.0/extra/zaz.desktop	2010-07-19 09:36:37.435671706 +0200
@@ -1,12 +1,10 @@
 [Desktop Entry]
-Version=1.0
 Type=Application
 Name=Zaz
 Comment=Zaz ain't Z***
+Comment[pl]=Zręcznościowa gra logiczna
 Comment[ru]=Zaz не клон Z***
 Comment[tr]=Top Yok Etme Oyunu
-TryExec=zaz
 Exec=zaz
-Categories=Game;LogicGame;
+Categories=Game;ArcadeGame;
 Icon=zaz
-MimeType=
