--- zaz-0.7.0/extra/zaz.desktop~	2010-05-01 00:06:55.000000000 +0200
+++ zaz-0.7.0/extra/zaz.desktop	2010-05-09 13:16:18.034800880 +0200
@@ -1,14 +1,11 @@
-#!/usr/bin/env xdg-open
 [Desktop Entry]
-Version=1.0
 Type=Application
 Name=Zaz
 Name[ru]=Zaz
 Comment=Zaz ain't Z***
+Comment[pl]=Zręcznościowa gra logiczna
 Comment[ru]=Zaz не клон Z***
 GenericName[tr]=Top Yok Etme Oyunu
-TryExec=zaz
 Exec=zaz
-Categories=Game;LogicGame;
+Categories=Game;ArcadeGame;
 Icon=zaz
-MimeType=
